Tests/test_SeqFeature.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2015 by Kai Blin.  All rights reserved.
+# This code is part of the Biopython distribution and governed by its
+# license.  Please see the LICENSE file that should have been included
+# as part of this package.
+
+"""Tests Bio.SeqFeature.
+"""
+import unittest
+from os import path
+from Bio import SeqIO
+from Bio.SeqFeature import Reference
+
+class TestReference(unittest.TestCase):
+    """Tests for the SeqFeature.Reference class"""
+
+    def test_eq_identical(self):
+        """Test two identical references eq() to True"""
+        testfile = path.join('GenBank', 'origin_line.gb')
+        rec1 = SeqIO.read(testfile, 'genbank')
+        rec2 = SeqIO.read(testfile, 'genbank')
+
+        self.assertEqual(rec1.annotations['references'][0], rec1.annotations['references'][0])
+        cmp1, cmp2 = rec1.annotations['references'][0], rec2.annotations['references'][0]
+        self.assertEqual(cmp1, cmp2, "%s vs %s" % (cmp1, cmp2))
```

Comment:
Just use ``self.assertEqual(cmp1, cmp2)`` which has a default message which will nicely display the non-equal elements. It handles lists etc nicely too which is a bonus.

This will also stop QualifiedCode nagging about old-style string formatting ;)